internal/model/matcher.go

```diff
@@ -22,3 +26,29 @@ func (c CompositePathMatcher) Matches(f string, isDir bool) (bool, error) {
 }
 
 var _ PathMatcher = CompositePathMatcher{}
+
+type singlePathMatcher struct {
+	repoRoot string
+	path     string
+}
+
+func (p singlePathMatcher) Matches(f string, isDir bool) (bool, error) {
```

Comment:
oh, i thought we were going to use the existing docker path matcher that uses golang path patterns?

Reply:
Yep good idea, I'll just use that which will simplify this PR quite a bit.